configuration/config-txt/overclocking.md

```diff
@@ -73,22 +73,23 @@ This table lists the default `over_voltage` settings for the various Pi models.
 | --- | --- | --- |
 | Pi 1 | 0 | 1.2V |
 | Pi 2 | 0 | 1.2-1.3125V |
-| Pi 3 | 0 | 1.2-1.3125V |
+| Pi 3 | 0 | 1.2-1.3125V 
+| Pi 4 | 0 | 0.88V |
 | Pi Zero | 6 | 1.35V |
 
 #### Specific to Pi 4B
 
-The `core_freq` of the Raspberry Pi 4 can change from the default if either `hdmi_enable_4kp60` or `enable_tvout` are used, due to relationship between internal clocks and the particular requirements of the requested display modes.
+The minimum core frequency when the system is idle must be fast enough to support the highest pixel clock (ignoring blanking) of the display(s). Consequently, `core_freq` will be boosted above 500 MHz if the display mode is 4Kp60.
 
-| Display option | Frequency |
+| Display option | Max `core_freq` |
 | -------------- | --------: |
 | Default        | 500 |
-| enable_tvout | 360 |
 | hdmi_enable_4kp60 | 550 |
 
-Changing `core_freq` in `config.txt` is not supported on the Pi 4, any change from the default will almost certainly cause a failure to boot.
-
-It is recommended when overclocking to use the individual frequency settings (`isp_freq`, `v3d_freq` etc) rather than `gpu_freq`, as since it attempts to set `core_freq` (which cannot be changed on the Pi 4), it is not likely to have the desired effect.
+* Overclocking requires the latest firmware release. Please make a backup before using [rpi-update](../../raspbian/applications/rpi-update.md)
```

Comment:
Does it require an rpi-update release? Do we want to be pointing people to rpi-update - if so a warning about it installing pre-release software would be useful.

Reply:
The warning on https://github.com/raspberrypi/documentation/blob/master/raspbian/applications/rpi-update.md is probably already sufficient?

Reply:
> The warning on https://github.com/raspberrypi/documentation/blob/master/raspbian/applications/rpi-update.md is probably already sufficient?

Good point. In which case the suggestion on this page to make a backup is not necessary.

Reply:
I'll just delete the second sentance